coordinator/coordinator.go

```diff
@@ -956,18 +950,13 @@ func (c *Coordinator) checkS3Import(featureProvider *metadata.Provider) (bool, e
 	return false, nil
 }
 
-func (c *Coordinator) runTrainingSetJob(resID metadata.ResourceID, schedule string) error {
+func (c *Coordinator) runTrainingSetJob(resID metadata.ResourceID, schedule string, runInfo jobInfo) error {
 	c.Logger.Info("Running training set job on resource: ", "name", resID.Name, "variant", resID.Variant)
 	ts, err := c.Metadata.GetTrainingSetVariant(context.Background(), metadata.NameVariant{Name: resID.Name, Variant: resID.Variant})
 	if err != nil {
 		return err
 	}
 
-	err = c.setPending(resID, ts.Status())
```

Comment:
Why are we removing these, sorry big PR trying to wrap my head about it

Reply:
Originally when resources were created they were put into a CREATED state and were set to PENDING when the job started. We've changed it so that now they're created in a PENDING state and when the job starts they have a RUNNING state. I moved where this is set outside of this function